tools/exports.js

```diff
@@ -1,15 +1,4 @@
-exports["Compressor"] = Compressor;
 exports["Dictionary"] = Dictionary;
-exports["JS_Parse_Error"] = JS_Parse_Error;
```

Comment:
I think some uglify 2.x wrappers need to access `JS_Parse_Error` for some reason. With the new 3.x minify API perhaps this is no longer an issue.